pkg/apis/monitoring/v1beta1/alertmanager_config_types.go

```diff
@@ -836,6 +838,19 @@ type TelegramConfig struct {
 	HTTPConfig *HTTPConfig `json:"httpConfig,omitempty"`
 }
 
+type MSTeamsConfig struct {
+	// Whether to notify about resolved alerts.
+	// +optional
+	SendResolved *bool `json:"sendResolved,omitempty"`
+	// MSTeams webhook url
+	WebhookUrl string `json:"webhookUrl"`
```

Comment:
If those fields are required, worth to add the `+required` annotation.

Reply:
I'm wondering if we could use `+kubebuilder:validation:Pattern` with a url regex on this, wdyt @xiu ?

Reply:
I think we could :)

```suggestion
	// +kubebuilder:validation:Pattern:="^http(s)?://.+$"
	// +required
	WebhookUrl string `json:"webhookUrl"`
```